certs/Swish_Merchant_TestCertificate_1234679304.key

```diff
@@ -0,0 +1,52 @@
+
+-----BEGIN RSA PRIVATE KEY-----
+MIIJKQIBAAKCAgEA4zcDDjX9HVzB4Pk2vVuhYfVZggUERpuig/hFol1f+6pYcQXQ
```

Comment:
These are certificates for the swish test environment that can be downloaded from the public swish documentation [here](https://developer.swish.nu/documentation/environments#certificates). However these do not match the current downloadable certificate, since I've reused the test certificates from the old Swedish api, but I'm very confident these are for testing.